oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -57,6 +58,8 @@
 public abstract class FulltextDocumentMaker<D> implements DocumentMaker<D> {
 
     private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final String WARN_LOG_STRING_SIZE_THRESHOLD_KEY = "oak.repository.property.logWarnStringSizeThreshold";
```

Comment:
Maybe make this field package private so that the test can simply use the variable instead of hard coding the value!?

Reply:
Changed access modifier to public.